CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Rust IPFS
+
+Welcome, and thank you for your interest in contributing to Rust IPFS. Issues and pull requests are encouraged.
+
+## Target Build
+
+Rust IPFS will always target the current _stable_ version of Rust that is released. Our CI/CD tests will reflect this.
+
+TODO: Instructions on installing the up-to-date stable toolchain
+TODO: First Principles
+
+## Roadmap
+
+- Link to roadmap issue or project?
+
+## Contributing
+
+Reporting Bugs
+Suggesting Enhancements
+Your First Code Contribution
+Pull Requests
+
+## TODO: Styleguides
+
+Git Commit Messages
+JavaScript Styleguide
+CoffeeScript Styleguide
+Specs Styleguide
+Documentation Styleguide
+
+## Security vuln disclosure - perhaps in SECURITY.md?
```

Comment:
I think this could be just clear policy to use issues, which I guess is the default and the one for rust-libp2p which doesn't have a CONTRIBUTING.md.